Documentation/energy-output.rst

```diff
@@ -1,27 +1,35 @@
 Energy output
-============
+==============
 
 Introduction
-------------
+--------------
 
 The energy output computes the energy of the simulation. It is divided into multiple parts:
 
-- Energy in the water layer (= acoustic medium).
-    - Gravitational energy
-    - Acoustic energy
-    - Acoustic kinetic energy
-- Energy in Earth
-    - Elastic kinetic energy
-    - Elastic energy
+- Energy in the water layer (= acoustic medium, :math:`\Omega_a`)
+    - Gravitational energy 
+        :math:`\int_{\Omega_a} \frac{1}{2} \rho g \eta^2 \,\mathbf{dx}`, with :math:`\rho` the density, :math:`g` the gravitational acceleration and :math:`\eta` the sea-surface elevation.
+    - Acoustic energy 
+        :math:`\int_{\Omega_a} \frac{1}{2K} p^2 \,\mathbf{dx}`, with :math:`p` the acoustic pressure and :math:`K` the compressibility.
+    - Acoustic kinetic energy 
+        :math:`\int_{\Omega_a} \frac{1}{2} \rho v^2 \,\mathbf{dx}`, with :math:`\rho` the density and :math:`v` the velocity.
+- Energy in Earth :math:`\Omega_e`
+    - Elastic kinetic energy 
+        :math:`\int_{\Omega_e} \frac{1}{2} \rho v^2 \,\mathbf{dx}`, with :math:`\rho` the density and :math:`v` the velocity.
+    - Elastic energy 
+        :math:`\int_{\Omega_e} \frac{1}{2} \sigma_{ij} c_{ijkl} \sigma_{kl} \,\mathbf{dx}`, with  :math:`\sigma_{ij}` the stress tensor and :math:`c_{ijkl}` the elastic moduli tensor relating stress and strain. It reduces for isotropic materials to :math:`\int_{\Omega_e} \frac{1}{2\mu} (\sigma_{ij} \sigma_{ij} -\frac{\lambda}{3\lambda+2\mu} \sigma_{kk}^2)\,\mathbf{dx}`, with :math:`\lambda` and :math:`\mu` the Lame coefficients.
```

Comment:
This one isn't correct, best to write this as \epsilon_ij \sigma_ij with epsilon being the strain. The c tensor here should be the *inverse* of the c tensor, which would convert stress to strain